rest/config.go

```diff
@@ -651,7 +651,8 @@ func decodeAndSanitiseConfig(r io.Reader, config interface{}) (err error) {
 	}
 
 	b = base.ConvertBackQuotedStrings(b)
-
+	// Expand environment variables
+	b = expandEnv(b)
```

Comment:
Can we move this up to run before ConvertBackQuotedStrings?

Reply:
Yeah, this can be moved up to run before ConvertBackQuotedStrings and it is included in the latest commit.